source/stylesheets/_refill.scss

```diff
@@ -0,0 +1,218 @@
+section.refill {
+  @include clearfix;
+  margin-bottom: 2em;
+  padding-bottom: 1em;
+  position: relative;
+
+  > h3 {
+    margin-bottom: 1.3em;
+    margin-top: 1em;
+    padding-top: 0.5em;
+  }
+}
+
+.refills-wrapper {
+  @include outer-container;
+  padding: 1em;
+
+  h1.main-header {
+    background-color: $refills-color;
+    display: inline-block;
+    padding: 0.5em;
+  }
+}
+
+.refill-centering {
+  text-align: center;
+
+  & > * {
+    display: inline-block;
+
+    & > * {
+      text-align: left;
+    }
+  }
+
+  script {
+    display: none;
+  }
+}
+
+.line-behind-text {
+  border-bottom: 1px solid #cacaca;
+  text-align: center;
+  margin-bottom: 4.5em;
+}
+
+.line-behind-text h6 {
+  background: #fff;
+  color: gray;
+  display: inline-block;
+  font-family: $refills-narrow-sans;
+  font-size: 1em;
+  font-weight: 500;
+  padding: 0 10px;
+  position: relative;
+  text-transform: uppercase;
+  top: 34px;
+}
+
+.refill h6 {
+  margin-bottom: 1.5em;
+
+}
+
+.js-hide-code,
+.js-show-code {
+  border-top: 1px dashed #bebcbc;
+  color: #7e7a7a;
```

Comment:
Color literals like `#7e7a7a` should only be used in variable declarations; they should be referred to via variable everywhere else.